resource/packages/apt/apt.go

```diff
@@ -35,14 +34,12 @@ type Package struct {
 func (p *Package) Check(resource.Renderer) (resource.TaskStatus, error) {
 	status := resource.NewStatus()
 
-	currentPkgStatusRaw, err := exec.Command("bash", "-c", "apt-cache policy "+p.Name+" | grep Installed | awk '{print $2}'").Output()
+	currentPkgStatusRaw, err := exec.Command("sh", "-c", "apt-cache policy "+p.Name+" | grep Installed | awk '{print $2}'").Output()
 	if err != nil {
-		log.WithField("module", "apt").Warn(err)
+		return status, errors.Wrapf(err, "checking package %s", p.Name)
 	}
 	currentPkgStatus := strings.TrimSpace(string(currentPkgStatusRaw))
 
-	log.WithField("module", "apt").WithField("exec", "bash").Info(string(currentPkgStatus))
-	log.Info(p)
 	if p.State == "installed" {
 		if string(currentPkgStatus) == "(none)" {
 			status.AddDifference(p.Name, string(currentPkgStatus), "installed", "")
```

Comment:
I haven't dug deeper: does `AddDifference()` set some kind of Status on its own?